authbridge/demos/finance-sparc/Makefile

```
@@ -0,0 +1,213 @@
+AGENT_OLLAMA_MODEL ?= llama3.2:3b
+endif
+
+help:  ## Show this menu
+	@printf "\nfinance-sparc demo — SPARC reflection on a regular finance agent.\n\n"
+	@printf "Run:\n  \033[1mmake demo\033[0m                 (watsonx; needs WX_* env)\n"
+	@printf "  \033[1mmake demo PROVIDER=ollama\033[0m  (local ollama; no creds)\n\n"
+	@printf "Targets:\n"
+	@awk 'BEGIN{FS=":.*## "} /^[a-zA-Z_-]+:.*## /{printf "  \033[36m%-16s\033[0m %s\n",$$1,$$2}' $(MAKEFILE_LIST)
+	@printf "\nVars: PROVIDER(=%s) KIND_CLUSTER_NAME AUTHBRIDGE_IMAGE SPARC_MODEL HOST_OLLAMA_URL\n\n" "$(PROVIDER)"
+
+preflight:  ## Verify kagenti + tools (+ watsonx creds when PROVIDER=watsonx)
```

checkmake (0.3.2)

[warning] Target body for "preflight" exceeds allowed length of 5 lines (6).

(maxbodylength)